scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -640,6 +665,55 @@ def test_magnitude_single_rotation():
     assert_allclose(result2, 0)
 
 
+def test_reduction_no_indices():
+    result = Rotation.identity().reduce(return_indices=False)
+    assert type(result) == Rotation
+
+
+def test_reduction_none_indices():
+    result = Rotation.identity().reduce(return_indices=True)
+    assert type(result) == tuple
+    assert len(result) == 3
+
+    reduced, left_best, right_best = result
+    assert left_best is None
+    assert right_best is None
+
+
+def test_reduction_full_indices():
```

Comment:
I think everything in this tests is essentially tested in the next test, maybe it's not necessary.

Reply:
Agreed.  Fixed.